orienteer-graph/src/main/java/org/orienteer/graph/component/widget/GraphEdgesWidget.java

```diff
@@ -51,7 +56,9 @@ public GraphEdgesWidget(String id, IModel<ODocument> model, IModel<ODocument> wi
         IModel<DisplayMode> modeModel = DisplayMode.VIEW.asModel();
         Form<ODocument> form = new Form<ODocument>("form");
 
-        VertexEdgesDataProvider vertexEdgesDataProvider = new VertexEdgesDataProvider(getModel());
+        OQueryDataProvider<ODocument> vertexEdgesDataProvider = new OQueryDataProvider<ODocument>("SELECT expand(bothE()) FROM "+model.getObject().getClassName()+" WHERE @rid=:rid");
+        vertexEdgesDataProvider.setParameter("rid", Model.of(model.getObject().getIdentity()));
```

Comment:
See below, but regarding this structure: commonly "unpacking of a model and then packing it to another model" is bad pattern.